airflow/executors/workloads.py

```diff
@@ -75,6 +77,15 @@ def key(self) -> TaskInstanceKey:
         )
 
 
+class DagRun(BaseModel):
+    id: int
+    dag_id: str
+    run_id: str
+    logical_date: datetime
+    data_interval_start: datetime
+    data_interval_end: datetime
```

Comment:
Looks like a lot of classes are duplicated across this file & `airflow/api_fastapi/execution_api/datamodels`.

We should figure this out with our packaging discussion if this entire file is going to be moved to  separate package.

Reply:
This one isn't needed with getting DR from context, removed it.